internal/commands/logs/travis.go

```diff
@@ -0,0 +1,13 @@
+package logs
+
+import (
+	"github.com/cirruslabs/echelon"
+)
+
+func NewTravisCILogsRenderer(renderer echelon.LogRendered) echelon.LogRendered {
```

Comment:
Does it work with any renderer or only specific ones like `SimpleRenderer` from `github.com/cirruslabs/echelon/renderers`?

If it's the latter, it would be cool to assert that somehow, or at least leave a comment.

Reply:
Change SimpleRendeder since need to make sure that a folding only starts once